yt_dlp/extractor/facebook.py

```diff
@@ -518,6 +541,8 @@ def extract_metadata(webpage):
             # in https://www.facebook.com/yaroslav.korpan/videos/1417995061575415/
             if thumbnail and not re.search(r'\.(?:jpg|png)', thumbnail):
                 thumbnail = None
+            profile_url = get_first(post, ('attachments', ..., 'media', 'creation_story', 'comet_sections', 'actor_photo', 'story', 'actors', ..., 'profile_url'))
+            follower_count = extract_follower_count(self._download_webpage(profile_url, None))
```

Comment:
we can inline `channel_follower_count` in the `info_dict` assignment below, and we're not using `profile_url` at all?
```suggestion
```